.docs/dev-guide/build-reference.md

```diff
@@ -4,7 +4,56 @@ How to build libStorage
 
 ---
 
-## Build Requirements
+## Basic Builds
+The following one-line command is the quickest, simplest, and most
+deterministic approach to building libStorage:
+
+```bash
+$ git clone https://github.com/codedellemc/libstorage && make -C libstorage
+```
```

Comment:
Hi @codenrhoden,

Thoughts on this type of note?

![image](https://cloud.githubusercontent.com/assets/101085/23237119/9ede8662-f921-11e6-8fb6-56491516385c.png)